specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
`summary_results_table` summarizes a Dominion report about overvotes, blanks, and valid votes. I guess in some sense that is Dominion's version of an audit report, though the term is confusing in our context. It certainly would be a useful thing for us to audit that report against the CVRs etc.
But I haven't seen any requirements around us dealing with their report. So I guess it is an opportunity for future work.